rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_write_interface.hpp

```diff
@@ -34,6 +34,8 @@ class ROSBAG2_STORAGE_PUBLIC BaseWriteInterface
   virtual ~BaseWriteInterface() = default;
 
   virtual void write(std::shared_ptr<const SerializedBagMessage> msg) = 0;
+  // Method for bulk writes based on transactions
```

Comment:
remove comment as this is implementation specific. Add a new line between the two functions though.